cmd/crc/cmd/daemon_darwin.go

```diff
@@ -30,13 +30,20 @@ func httpListener() (net.Listener, error) {
 }
 
 func checkIfDaemonIsRunning() (bool, error) {
+	if launchd.PlistExists(constants.DaemonAgentLabel) {
+		/* detect if the daemon is being started by launchd,
+		* and socket activation is in use. In this scenario,
```

Comment:
Since socket activation is not being used with launchd, we don't need this code?